common/src/main/java/tachyon/worker/DataServerMessage.java

```diff
@@ -166,9 +161,10 @@ public static DataServerMessage createBlockResponseMessage(boolean toSend, long
 
   private RPCResponse.Status mStatus;
 
-  // TODO(calvin): Investigate how to remove this since it is not transferred over the wire.
   private long mLockId = -1L;
 
+  private long mSessionId;
```

Comment:
I am not sure about the use of this mLockId before, so to be safe I just left it as is.